pkg/ui/query.go

```diff
@@ -122,7 +123,28 @@ func (q *Query) stores(w http.ResponseWriter, r *http.Request) {
 	for _, status := range q.storeSet.GetStoreStatus() {
 		statuses[status.StoreType] = append(statuses[status.StoreType], status)
 	}
-	q.executeTemplate(w, "stores.html", prefix, statuses)
+
+	sources := make([]component.StoreAPI, 0, len(statuses))
+	for k := range statuses {
+		sources = append(sources, k)
+	}
+	sort.Slice(sources, func(i int, j int) bool {
+		if sources[i] == nil {
+			return false
+		} else if sources[j] == nil {
```

Comment:
You never need `else` if you are returning, let's simplify this (:

Reply:
not done?

Reply:
Better ?